tee-launcher/launcher-docker-compose.yaml

```
@@ -2,16 +2,27 @@ version: '3.8'
 
 services:
   web:
-    image: barakeinavhnear/launcher:latest # todo (security) use spesific image hash  
+    image: barakeinavhnear/launcher:latest  # TODO (security): Replace with a specific image digest
     container_name: launcher
+
     environment:
       - DOCKER_CONTENT_TRUST=1
-      - DEFAULT_IMAGE_DIGEST=sha256:4b08c2745a33aa28503e86e33547cc5a564abbb13ed73755937ded1429358c9d # nearone/mpc-node-gcp:testnet-release
+      - DEFAULT_IMAGE_DIGEST=sha256:4b08c2745a33aa28503e86e33547cc5a564abbb13ed73755937ded1429358c9d  # nearone/mpc-node-gcp:testnet-release
+
     volumes:
-      - /var/run/docker.sock:/var/run/docker.sock
+      - /var/run/docker.sock:/var/run/docker.sock 
```

Please make sure you are using an editor that properly formats the code and removes trailing whitespaces.
```suggestion
      - /var/run/docker.sock:/var/run/docker.sock
```

---

thanks.